README.md

```diff
@@ -808,6 +808,28 @@ By passing a special prop `adyenConfiguration` we are giving a possibility to cu
 }
 ```
 
+### <a id="security"></a><h2>Security</h2>
```

Comment:
let's add a header and link at the top of the readme (next to translations/styling/adyen config etc)